napari/layers/_labels_layer/model.py

```diff
@@ -0,0 +1,226 @@
+import numpy as np
+from numpy import clip
+from copy import copy
+
+from .._base_layer import Layer
+from ..._vispy.scene.visuals import Image as ImageNode
+
+from ...util.colormaps import colormaps
+from ...util.event import Event
+
+from .._register import add_to_viewer
+
+from .view import QtLabelsLayer
+
+
+@add_to_viewer
+class Labels(Layer):
+    """Image layer.
```

Comment:
I think the docstring should be updated to be more descriptive.

Reply:
Done!